src/Counter/CounterForm.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import ButtonList from "./ButtonList";
+
+class CounterForm extends React.Component {
```

Comment:
I don't see any form here, seems like it's just `Counter`

Reply:
fixed